### PR TITLE
Update configs to work for prod & cleanup

### PR DESCRIPTION
**Pull request by teghnet**
This is in preparation for prod values.

It also removes legacy configs.

### CODE REVIEW

File: cmd/ghost/main.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ package main
 import (
 	"os"
 
+	suite "github.com/chronicleprotocol/oracle-suite"
 	"github.com/chronicleprotocol/oracle-suite/cmd"
```
Comment (WesleyCharlesBlake) [inline cmd/ghost/main.go:22]: Does `../oracle-suite/cmd` not need to be removed as well? as per changes made in the -ldflags in the Dockerfile?

File: config-ghost.hcl [hunk 2]
```diff
@@ -1,8 +1,7 @@
 ghost {
   ethereum_key = "default"
-
-  interval = try(tonumber(env.CFG_GHOST_INTERVAL), 60)
-  pairs    = try(env.CFG_GHOST_PAIRS == "" ? [] : split(",", env.CFG_GHOST_PAIRS), [
+  interval     = try(tonumber(env.CFG_GHOST_INTERVAL), 60)
+  data_models  = [
```
Comment (WesleyCharlesBlake) [inline config-ghost.hcl:4]: just to be clear, are we changing 'pairs[]` to `data_models` ?

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,14 +1,15 @@
 version: '3'
 services:
-  bootstrap:
+  boot:
```
Comment (WesleyCharlesBlake) [inline docker-compose.yaml:3]: is `boot` any different to `bootstrap`? 

TBO: `bootstrap` is more intuitive, so unless theres another valid reason to rename this?

**Review by WesleyCharlesBlake — APPROVED** [review]
just some questions left in the review

File: docker-compose.yaml (hunk 3 above)
Comment (teghnet) [reply docker-compose.yaml:3]: no valid reason, just wanted it to be shorter.

File: config-ghost.hcl (hunk 2 above)
Comment (teghnet) [reply config-ghost.hcl:4]: Yes, but it's only internal.

**Review by WesleyCharlesBlake — APPROVED** [review]

**Pull request merged**